dex/networks/eth/params.go

```diff
@@ -197,3 +187,61 @@ type Redemption struct {
 	Secret     [32]byte
 	SecretHash [32]byte
 }
+
+var testTokenID, _ = dex.BipSymbolID("dextt.eth")
+
+var Tokens = map[uint32]*dex.Token{
+	// testTokenID = 'dextt.eth' is the used for the test token from
+	// dex/networks/erc20/contracts/TestToken.sol that is deployed on the simnet
+	// harness, and possibly other networks too if needed for testing.
+	testTokenID: {
+		NetAddresses: map[dex.Network]*dex.TokenAddresses{
+			dex.Mainnet: {
+				Address:       common.Address{},
+				SwapContracts: map[uint32][20]byte{},
+			},
+			dex.Testnet: {
+				Address:       common.Address{},
+				SwapContracts: map[uint32][20]byte{},
+			},
+			dex.Simnet: {
+				Address:       common.Address{},
+				SwapContracts: map[uint32][20]byte{},
+			},
+		},
+		ParentID: EthBipID,
+		Name:     "DCRDEXTestToken",
+		UnitInfo: dex.UnitInfo{
+			AtomicUnit: "Dextoshi",
+			Conventional: dex.Denomination{
+				Unit:             "DEXTT",
+				ConversionFactor: GweiFactor,
+			},
+		},
+		Gas: dex.Gases{
+			Swap:      157_000,
+			SwapAdd:   115_000,
+			Redeem:    70_000,
+			RedeemAdd: 33_000,
+			Refund:    50_000,
+			Approve:   29_000,
+			Transfer:  31_000,
+		},
+	},
+}
+
+func VersionedNetworkToken(assetID uint32, contractVer uint32, net dex.Network) (*dex.Token, *dex.TokenAddresses, common.Address, error) {
```

Comment:
Why do you need to return both the `dex.TokenAddresses` and also the specific address for the version requested?